drift_dev/lib/src/analysis/resolver/drift/table.dart

```diff
@@ -72,6 +73,9 @@ class DriftTableResolver extends LocalElementResolver<DiscoveredDriftTable> {
             (msg) => reportError(
                 DriftAnalysisError.inDriftFile(column.definition ?? stmt, msg)),
             dartClass.classElement.thisType,
+            column.type.type == BasicType.int
+                ? EnumType.intEnum
+                : EnumType.textEnum,
```

Comment:
Is it the correct way to do this conditional check?

Reply:
Let's use the local `type` variable instead. You can compare it against `DriftSqlType.int`.

Reply:
I've used `type` in [🎨 Better type comparison and matcher](https://github.com/simolus3/drift/pull/2168/commits/4030db4c5f6eb7795f3f21c2a65065f025bdbdc0)